core.md

```diff
@@ -0,0 +1,131 @@
+# Governance of the NeoForged project
+
+The NeoForged project is split between [GitHub](https://github.com/NeoForged) and [Discord](https://discord.neoforged.net). It is a community of volunteers, maintaining a free and open source ecosystem of modifications to the Minecraft game.
+
+All contributors and team members are here for fun - our goal is to provide joy to the community, and to ourselves. 
+
+We are here because it is fun, so this document ensures that the Project doesn't become so bureaucratic or so much of a chore that the fun is removed for anybody.
+
+### What this document is
+
+This document is:
+ * A non-exhaustive list of the working principles of the project.
+ * Intended to help guide the project in times of uncertainty.
+
+This document is built on the fundamental principle of "people generally want to help, so we should not get in the way of it".
+
+The core ethic is "people over processes", so this document is intentionally light on processes - situations that come up are rarely one-size-fits-all, and should be treated on a case by case basis, with the best process to fit the person.
+
+### What this document is not
+
+This document is not:
+ * Intended to enforce rules on all members.
+ * Intended to catch all possibilities.
+ * Intended to prevent "rules lawyering".
+
+The Code of Conduct of the project is taken to be the content of [the Contributor Covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/), so this document is not the Code of Conduct. However, this may be altered with a copy held and maintained by the Project, only with overwhelming majority support by the entire team.
```

Comment:
I am against the adoption of the contributor covenant due to its usage of politically contentious language and associated concerning FAQ, but can be on board with an altered version.

Reply:
What are you referring to in particular?

Reply:
I agree. Contributors covenant is a widely accepted document for operating within public online spheres. It's only "political" controversy is from those who wish not to be bound by some of it's terms.